### ISSUE TITLE
CSC deployment feedback 

### ISSUE THREAD

**Issue by erikvdbergh** [EMBL-EBI-TSI/TESK#23]
Relevant comments below:

- I happen to know that these are Jinja2 templates and I can refresh my memory
  on how to compile Jinja2 templates based on an ini file, but this could be
  documented as well as it may not be obvious.

- And reading a bit further it looks like j2cli doesn't work with fileglobs, so
  install shinto-cli after all and use that.

- I guessed what the correct values for settings would be in config.ini. I had
  previously created a namespace called "tesk" for my testing so I used that for
  the namespace settings.

- ~~I also tried the example request in README.md (I had to modify it a bit):
  curl -H "Content-Type: application/json" -d @specs/examples/task_example.json
  https://tesk-test.rahtiapp.fi/v1/tasks~~

- ~~The command above gave me an HTTP 400 error with:
  JSON parse error: Unrecognized field \"project\"~~

Tasks:
- [x] point user to use shinto-cli in docs to use the jinja templates
- [x] comment in config.ini what default values could be and should be
- [x] examples should work
- [x] point to a first example to run from the examples folder

**Comment by giannidallatorre** [comment]
Relevant comments below:

- I stumbled on the deployment documentation. The deployment documentation
was clear once I found the correct file.

My comment, I just waiting for the confirmation to that `shinto-cli` works on linux and windows, then I will clean up the old documentation and link the new one.

I should have done a branch with the restructure documentation using templates, but I was not expecting that beta tester on this stage without any informal instruction, my mistake
